Userland/Libraries/LibWeb/Loader/GeneratedPagesLoader.cpp

```diff
@@ -89,7 +89,7 @@ ErrorOr<String> load_file_directory_page(LoadRequest const& request)
     StringBuilder builder;
     SourceGenerator generator { builder };
     generator.set("path", escape_html_entities(lexical_path.string()));
-    generator.set("parent_path", escape_html_entities(lexical_path.parent().string()));
+    generator.set("parent_url", TRY(String::formatted("file://{}", lexical_path.parent().string())));
```

Comment:
Are we sure we don't need to `escape_html_entities` any longer?

Reply:
No because this variable is only used in href attr

Reply:
Why's that matter? That was also the case before this patch. If the string contains a quote or an opening/closing brace, we would generate invalid HTML.

Reply:
Sure, my thoughts where then the URL is invalid but yea, adding it back won't hurt